include/extension-api.h

```diff
@@ -62,8 +63,23 @@ class Engine {
   void enable_module_mode(bool enable);
   bool eval_toplevel(const char *path, MutableHandleValue result);
 
+  /**
+   * Run the JS task queue and wait on pending tasks until there
+   * are no outstanding lifetimes to wait on.
+   */
   bool run_event_loop();
```

Comment:
I find `lifetime` pretty confusing as a term here. How would you feel about something like `interest`? With this comment modified like this, and other places (in code and comments) accordingly:
```suggestion
  /**
   * Run the async event loop as long as there's interest registered in keeping it running.
   *
   * Each turn of the event loop consists of three steps:
   * 1. Run reactions to all promises that have been resolves/rejected.
   * 2. Check if there's any interest registered in continuing to wait for async tasks, and terminate the loop if not.
   * 3. Wait for the next async task to become ready and execute it.
   *
   * Interest or loss of interest in keeping the event loop running can be signaled using the `Engine::raise_event_loop_interest` and `Engine::lower_event_loop_interest` methods.
   *
   * If no async tasks remain pending while there's still interest in the event loop, an error will be reported.
   */
  bool run_event_loop();
```

Reply:
Renamed to interest, and added docs.